tests/functional/test_stub.py

```diff
@@ -148,3 +150,19 @@ def test_still_relies_on_param_validation_with_expected_params(self):
         # Throw an error for invalid parameters
         with self.assertRaises(ParamValidationError):
             self.client.list_objects(Buck='bar')
+
+    def test_any_ignores_param_for_validation(self):
```

Comment:
Can we add a test with multiple params?  One stub.ANY and one with an expected value and verify you're allowed to mix the two types in the same response call?